sponsorship_compassion/wizards/generate_gift_wizard.py

```diff
@@ -134,15 +137,15 @@ def _setup_invoice_line(self, contract):
         return inv_line_data
 
     @api.model
-    def compute_date_birthday_invoice(self, child_birthdate, payment_date):
+    def _compute_date_birthday_invoice(self, child_birthdate, payment_date):

```

Comment:
Don't put it private, let it public to avoid warnings of calling a private method (rename all your calls as well)